src/@next/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { Story, Meta } from '@storybook/react';
+import { BaseContainer } from '../../Layout/GlintsContainer/GlintsContainer';
+import { TextArea } from './TextArea';
+import { TextAreaProps } from './TextArea';
+
+export default {
+  title: '@next/TextArea',
+  component: TextArea,
+  decorators: [Story => <BaseContainer>{Story()}</BaseContainer>],
+  argTypes: {},
+} as Meta;
+
+const Template: Story<TextAreaProps> = args => {
+  const [value, setValue] = useState<string>('');
+
+  return (
+    <>
+      <TextArea {...args} value={value} onChange={val => setValue(val)} />
+    </>
+  );
+};
+export const Interactive = Template.bind({});
+Interactive.args = {
+  placeholder: 'Please enter...',
+  disabled: false,
+  rows: 3,
+  maxLength: 60,
+  width: 520,
+};
+
+const WithPopulatedValueTemplate: Story<TextAreaProps> = args => {
```

Comment:
@ninariccimarie  just want to inform that removing this will also remove one snapshot from test case (the one with populated value); I created this example actually just purely for testing (as in Figma this variant is shown), as the `value` prop is used in useState I can't just add query params like `&args=value:abcde` to get the snapshot of populated text area, lmk if u have other alternative of testing this, or for now I'll remove the snapshot

Reply:
i see. you can add a test to fill the textArea. https://playwright.dev/docs/api/class-elementhandle#element-handle-fill